src/main/java/com/epam/university/java/core/task003/FilteringConditionImpl.java

```diff
@@ -0,0 +1,11 @@
+package com.epam.university.java.core.task003;
+
+public class FilteringConditionImpl implements FilteringCondition {
+    @Override
+    public boolean isValid(String element) {
+        if(element=="One"|element=="Two"|element==null){
```

Comment:
It's better to compare literals with equals, ex. "One".equals(element). And it is preferably to use short form of disjunctive operation.